### PR TITLE
Create fsdp-simple.yaml

### PR DESCRIPTION
**Pull request by KeitaW**
Add standalone CPU DDP example for EKS

Maybe we should update README as well.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: 3.test_cases/16.pytorch-cpu-ddp/kubernetes/fsdp-simple.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                - --checkpoint_path=/local/snapshot.pt
+
+              volumeMounts:
+                - name: shmem
+                  mountPath: /dev/shm
+                - name: local
+                  mountPath: /local
+          initContainers:
+            - name: script-downloader
+              image: public.ecr.aws/hpc-cloud/nccl-tests:latest
```
Comment (mhuguesaws) [inline 3.test_cases/16.pytorch-cpu-ddp/kubernetes/fsdp-simple.yaml:112]: Specify version. Latest is not a good practice.

File: 3.test_cases/16.pytorch-cpu-ddp/kubernetes/fsdp-simple.yaml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+              hostPath:
+                path: /mnt/k8s-disks/0
+          #nodeSelector:
+          #  node.kubernetes.io/instance-type: "${INSTANCE_TYPE}"
+          containers:
+            - name: pytorch
+              image: pytorch/pytorch:latest
+              imagePullPolicy: Always
+              command: 
+                - /opt/conda/bin/torchrun
```
Comment (mhuguesaws) [inline 3.test_cases/16.pytorch-cpu-ddp/kubernetes/fsdp-simple.yaml:96]: Why not mpirun operator? Not blocking, just asking.

**Review by mhuguesaws — APPROVED** [review]
Left comments.

**Review by shimomut — APPROVED** [review]
Approving.
Verified that I can run this manifest file in a workshop test account.

File: 3.test_cases/16.pytorch-cpu-ddp/kubernetes/fsdp-simple.yaml (hunk 2 above)
Comment (KeitaW) [reply 3.test_cases/16.pytorch-cpu-ddp/kubernetes/fsdp-simple.yaml:96]: Since we want to use it for `PyTorchJob` validation.

**Pull request merged**